### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by mst3k4ever**
Proposed edits.

### CODE REVIEW

File: _content/overview.md [hunk 1]
```diff
@@ -1,100 +1,103 @@
...
-- Be brief.
-- Oxford comma: In a list of three or more items, add a comma before the conjunction (for example: "Android, iOS, and Windows").
-- Spell check your content.
-- Be friendly by using "you". And remember that Agora can "recommend" courses of action.
-- Review your content. Edit out any information your reader does not need to know.
-- Use parallel structures to tell the reader how related concepts are different and how they are similar.
-- Place essential information in the main clause, as you cannot assume the reader will read the whole sentence.
-- Remove ambiguity by choosing words whose meaning is clear.
-- Use the relative pronoun "that" where you might otherwise omit it, and prefer using it to a noun-...ing combination, for example say "the method that uses this parameter" instead of "the method using this parameter".
+Best practices to adopt: //TODO William Are there too many? Should we simplify this or break it up? If you wanted something someone could maybe print (on one page) and put up by their workstation, this might be too much.
```
Comment (wangjie-agora) [inline _content/overview.md:46]: Agreed. Maybe we can keep only the most important principles.

File: _content/overview.md [hunk 2]
```diff
@@ -1,100 +1,103 @@
...
+
+* Create an outline to plan what to write before you start.
+* Always keep your audience's capabilites and needs in mind when you write.
+* Write for scannability by using bullet points, short paragraphs, and informative section headings to break up your content.
+* Keep paragraphs short. Dense pages full of text can be intimidating for readers (or hide important information from skimmers).
+* Remember to be clear and concise. Replace phrases with single words where possible, and remove ambiguities.
```
Comment (wangjie-agora) [inline _content/overview.md:52]: This one is repeating the previous sections.

File: _content/overview.md [hunk 3]
```diff
@@ -1,100 +1,103 @@
...
+* Favor the active voice.
+* Maintain a formal tone (except for FAQs). //TODO WIlliam I think?
+* Write in the second person ("you"), because it is more friendly and engaging than a third-person form (such as "developers"). Do not use "we," but remember that "Agora" can recommend courses of action. 
+* Always be consistent in your word usage. Do not use the same word to mean different things (or refer to one thing using different words).
+* Place essential information in the main clause, as you cannot assume the reader will read the whole sentence.
+* Put conditional clauses first. Start by mentioning the objective, followed the instruction (for example, "To run the program, click Run.").
+* Use the Oxford comma. That is, in a list of three or more items, add a comma before the conjunction (for example. "Android, iOS, and Windows").
+* Use parallel structures to tell the reader how related concepts are different and how they are similar.
+* Vary the length of your sentences.
```
Comment (wangjie-agora) [inline _content/overview.md:61]: Maybe put this together with the "Keep paragraphs short" one? Or just delete it?

File: _content/overview.md [hunk 4]
```diff
@@ -1,100 +1,103 @@
...
+Our readers come to our website with different abilities. Some may have issues with their eyesight and magnify the content on their screen. Others may use a screen reader that speaks the text on a web page. For some, English is their second language, and their reading comprehension level may be low. Content aimed at U.S. readers should be in line with the Americans with Disabilities Act (ADA) and use the [W3C Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) as a guide. By following the best practices outlined previously, you will already be doing most of what is required, but also keep the following in mind when you write:
```
Comment (wangjie-agora) [inline _content/overview.md:81]: We have discussed about the usage of "may" and "might", and decided to follow Google's style guide and use might to refer to possibilities.

**Review by wangjie-agora — COMMENTED** [review]
Hi Bill, thanks for the edits. I've replied to the TODOs and added some comments.

File: _content/overview.md [hunk 5]
```diff
@@ -1,100 +1,103 @@
...
+    It is more considerate to compile a complete list of your questions about a document before asking for help; it is less intrusive to interrupt someone once (even if it takes longer) than to keep coming back to ask multiple "quick questions."
+```
 
 #### Correct
 
-Our content must be technically and grammatically correct.
+Our content must be factually and grammatically correct.
 
-Incorrect content causes our audience to lose trust in our content and brand. If you are unsure if your content is technically correct, check with a developer before publication.
+Incorrect content causes our audience to lose trust in our documentation and brand. If you are unsure if your content is correct, check with a developer before publication. //TODO William Is there not a technical review at some point?
```
[the remark this answers is not in the document]
Comment (mst3k4ever) [reply _content/overview.md:38]: Okay, that's fine.

File: _content/overview.md [hunk 6]
```diff
@@ -1,100 +1,103 @@
...
-- Use the relative pronoun "that" where you might otherwise omit it, and prefer using it to a noun-...ing combination, for example say "the method that uses this parameter" instead of "the method using this parameter".
+Best practices to adopt: //TODO William Are there too many? Should we simplify this or break it up? If you wanted something someone could maybe print (on one page) and put up by their workstation, this might be too much.
+
+* Create an outline to plan what to write before you start.
+* Always keep your audience's capabilites and needs in mind when you write.
+* Write for scannability by using bullet points, short paragraphs, and informative section headings to break up your content.
+* Keep paragraphs short. Dense pages full of text can be intimidating for readers (or hide important information from skimmers).
+* Remember to be clear and concise. Replace phrases with single words where possible, and remove ambiguities.
+* Favor the active voice.
+* Maintain a formal tone (except for FAQs). //TODO WIlliam I think?
```
[the remark this answers is not in the document]
Comment (mst3k4ever) [reply _content/overview.md:54]: Okay!

File: _content/overview.md (hunk 1 above)
Comment (mst3k4ever) [reply _content/overview.md:46]: Okay. I will review the section again.

File: _content/overview.md (hunk 2 above)
Comment (mst3k4ever) [reply _content/overview.md:52]: I think there will be a lot of deletion and compression in this section. I have seen a few good examples of short "cheat sheets".

File: _content/overview.md (hunk 4 above)
Comment (mst3k4ever) [reply _content/overview.md:81]: Agreed. I will fix this.

File: _content/overview.md (hunk 3 above)
Comment (mst3k4ever) [reply _content/overview.md:61]: Yes, as part of the general consolidation this should get cleaned up.

**Comment by mst3k4ever** [comment]
I think this is done?

**Pull request closed without merging**